package.json

```diff
@@ -46,9 +50,5 @@
         "*": [
             "git add"
         ]
-    },
-    "checkLicense": [
-        "examples",
-        "force-app"
-    ]
+    }
```

Comment:
The checkLicense script is better done in our Jenkins pipeline since consumers of this repo probably wouldn't want a Salesforce header in their files.